share/nisar/defaults/static.yaml

```
@@ -0,0 +1,436 @@
+        # surface, $A_\beta$.
+        # Must be one of:
+        #   - 'pixel_area': uses the pixel area, which is the product of the range and
+        #     azimuth spacing (computed using the ground velocity)
+        #   - 'projection_angle': uses the projection angle ($\psi$):
+        #     $A_\beta = A_\sigma * \cos(\psi)$
+        #   - 'auto': the area beta mode is chosen by the software
+        # Defaults to 'auto'.
+        area_beta_mode: auto
+        # [OPTIONAL] Minimum pixel area threshold for RTC, in dB.
+        # If set to NaN, no minimum is enforced, which may lead to division by very
+        # small values and result in infinite or inaccurate outputs.
+        # Defaults to NaN.
+        min_area_factor: .NAN
+        # [OPTIONAL] DEM upsampling factor used for RTC.
+        # Defaults to 2.
+        dem_upsample_factor: 2.0
+
+      geo2rdr:
+        # [OPTIONAL] Absolute azimuth time convergence tolerance, in seconds, for the
+        # root-finding procedure in geo2rdr.
+        # Must be >= 0.
+        # Defaults to 1e-7.
+        threshold: 1e-7
+        # [OPTIONAL] Maximum number of Newton-Raphson iterations used by the
+        # root-finding procedure in geo2rdr.
+        # Must be >= 1.
+        # Defaults to 50.
+        maxiter: 50
+
+      rdr2geo:
+        # [OPTIONAL] Absolute slant range convergence tolerance, in meters, for the
+        # root-finding procedure in rdr2geo.
+        # Must be >= 0.
+        # Defaults to 1e-7.
+        threshold: 1e-7
+        # [OPTIONAL] Maximum number of primary Newton-Raphson iterations used by the
+        # root-finding procedure in rdr2geo.
+        # Must be >= 1.
+        # Defaults to 25.
+        numiter: 25
+        # [OPTIONAL] Maximum number of secondary iterations used by the root-finding
+        # procedure in rdr2geo.
+        # Must be >= 1.
+        # Defaults to 10.
+        extraiter: 10
+
+    output:
+      dataset:
+        # [OPTIONAL] Chunk dimensions of 2-D raster datasets in the output product.
+        # Setting `chunk_size` to [-1, -1] will disable chunked storage.
+        # Defaults to [512, 512].
+        chunk_size: [512, 512]
+        # [OPTIONAL] Enable HDF5 compression of raster datasets in the output product.
+        # Has no effect if chunked storage is disabled (see `chunk_size`).
+        # Defaults to true.
+        compression_enabled: true
+        # [OPTIONAL] HDF5 compression filter.
+        # Typically one of {'gzip', 'lzf', 'szip'}. See
+        # https://docs.h5py.org/en/stable/high/dataset.html#filter-pipeline for details.
+        # Ignored if `compression_enabled` is false.
+        # Defaults to 'gzip'.
+        compression_type: gzip
+        # [OPTIONAL] Level of compression applied by the GZIP compression filter.
+        # Must be an integer from 0 to 9 (inclusive).
+        # Ignored if `compression_enabled` is false or `compression_type` is not 'gzip'.
+        # Defaults to 4.
+        compression_level: 4
```

```suggestion
        compression_level: 1
```
I'd suggest changing the `compression_level` to `1` as this can reduce runtime without impacting too much the file size. We use `compression_level` equal `1` for [GCOV](https://github-fn.jpl.nasa.gov/isce-3/isce/blob/94bedfb2e7a94c8c0b84056ae3ee9f62d50d036c/share/nisar/defaults/gcov.yaml#L107) and [GSLC](https://github-fn.jpl.nasa.gov/isce-3/isce/blob/94bedfb2e7a94c8c0b84056ae3ee9f62d50d036c/share/nisar/defaults/gslc.yaml#L99).